test/parallel/test-cli-syntax.js

```diff
@@ -0,0 +1,55 @@
+var assert = require('assert');
```

Comment:
Can you add `'use strict'` here and use `const` for the imports?  Didn't `make test` complain about that?